src/vs/editor/common/services/languageFeatureDebounce.ts

```diff
@@ -75,7 +75,7 @@ class FeatureDebounceInformation implements IFeatureDebounceInformation {
 			this._cache.set(key, avg);
 		}
 		const newValue = clamp(avg.update(value), this._min, this._max);
-		if (!matchesScheme(model.uri, 'output')) {
+		if (!matchesScheme(model.uri, 'output') && this._logService) {
```

Comment:
This shouldn't be needed.

Reply:
It was throwing previously errors during the Unit tests on Linux when I pushed the code. I will check again if it works without.

Reply:
ok, otherwise we need to tweak how the tests are being setup. There is a `NullLogService` that would need to be added to the service collection. I can help with that, let me know if run into trouble